packages/react-aria-components/docs/GridList.mdx

```diff
@@ -361,6 +362,13 @@ import {GridList, Item, Checkbox, Button} from 'react-aria-components';
   <Selection />
 </ExampleCard>
 
+<ExampleCard
+  url="dnd.html"
+  title="Drag and drop"
+  description="Concepts and interactions for an accessible drag and drop experience.">
+  <DragAndDrop />
```

Comment:
Is showing the mouse cursor in here weird? Will people be confused and think their mouse is over there? Not showing it also looked weird because a drag is occurring...

Reply:
could use a different cursor (ie 'grabbing') or tint it blue to alleviate the effect? I don't find it too distracting though, it doesn't move when I move, so it's pretty quickly obvious which one is mine

Reply:
I think a blue tint/border would help.

Reply:
+1 to reid's comment

Reply:
unfortunately it doesn't look very good in dark mode when I do that. We don't have a dark enough blue in the variables.